client/my-sites/plugins/main.jsx

```diff
@@ -429,17 +433,22 @@ const PluginsMain = React.createClass( {
 export default connect(
 	state => {
 		const selectedSite = getSelectedSite( state );
-
+		const selectedOrAll = getSelectedOrAllSites( state );
```

Comment:
The _selected or all_ type of operations is idiomatic of old SitesList, but less so of all-Redux Calypso, where we can just grab `getSelectedSiteId( state )` and infer that `null` means "no site selected → all sites". Something to bear in mind as you work on more pieces towards ridding this module of SitesList!